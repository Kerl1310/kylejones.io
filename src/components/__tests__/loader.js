import React from 'react'
import renderer from 'react-test-renderer'

import Loader from '../loader'

describe('Loader', () => {
  it('renders correctly', () => {
    const tree = renderer.create(<Loader className="my-loader" />).toJSON()
    expect(tree).toMatchSnapshot()
  })
})
