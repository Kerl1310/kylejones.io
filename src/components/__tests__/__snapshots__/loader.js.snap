// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Loader renders correctly 1`] = `
<div
  className="sc-bdVaJa lbHbBB my-loader"
>
  <div />
  <div />
  <div />
  <div />
</div>
`;
